data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -324,6 +383,17 @@ bootkube_podman_run \
 	"${CLUSTER_BOOTSTRAP_IMAGE}" \
 	start --tear-down-early=false --asset-dir=/assets --required-pods="openshift-kube-apiserver/kube-apiserver,openshift-kube-scheduler/openshift-kube-scheduler,openshift-kube-controller-manager/kube-controller-manager,openshift-cluster-version/cluster-version-operator"
 
+if [ ! -z "$CLUSTER_ETCD_OPERATOR_IMAGE" ]
+then
+	echo "Waiting for ceo to finish..."
+	bootkube_podman_run \
+		--volume /etc/kubernetes:/etc/kubernetes:z \
+		"${CLUSTER_ETCD_OPERATOR_IMAGE}" \
+		/usr/bin/cluster-etcd-operator \
+			wait-for-ceo \
+			--kubeconfig /etc/kubernetes/kubeconfig
```

Comment:
maybe use the /assets/auth/kubeconfig ?? instead of the /etc/kubernetes/kubeconfig

Reply:
good catch